---a/net/minecraft/entity/IProjectile.java
+++b/net/minecraft/entity/IProjectile.java
@@ -1,7 +1,7 @@
 package net.minecraft.entity;
 
-public interface IProjectile
-{
+public interface IProjectile {
+
     void setThrowableHeading(double x, double y, double z, float velocity, float inaccuracy);
-}
+}
